src/main/flight/failsafe.h

```diff
@@ -18,6 +18,7 @@
 #pragma once
 
 #define FAILSAFE_POWER_ON_DELAY_US (1000 * 1000 * 5)
+#define MAX_COUNTER_VALUE_WHEN_RX_IS_RECEIVED_AFTER_RX_CYCLE 10
```

Comment:
Reusing constants in tests is generally a bad idea.  Tests should ideally be isolated from production code, e.g.  if i change test code production code should fail and vice versa.